lib/common.py

```diff
@@ -100,9 +100,15 @@ def call(command, withSlashLogger = True):
     logger = lambda x: None
 
   def readproc(proc):
-    for line in iter(proc.stdout.readline, ''):
-      readproc.output += line
-      logger(line.rstrip('\n'))
+    import sys
+    for line in iter(proc.stdout.readline, b''):
+      # check python version
```

Comment:
Why?
What behavior are you seeing when sharing the same code with both versions?

If we create Popen object with universal_newlines=True (i.e. it opens stdout in text mode), will shared code work for both versions?